shell/platform/darwin/ios/framework/Source/flutter_main_ios.mm

```diff
@@ -12,7 +12,18 @@ void FlutterMain() {
   NSBundle* bundle = [NSBundle bundleForClass:[FlutterViewController class]];
   NSString* icuDataPath = [bundle pathForResource:@"icudtl" ofType:@"dat"];
   NSString* libraryName = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"FLTLibraryPath"];
-  shell::PlatformMacMain(icuDataPath.UTF8String, libraryName != nil ? libraryName.UTF8String : "");
+
+  NSBundle* mainBundle = [NSBundle mainBundle];
+
+  NSString* flutterAssetsPath = [mainBundle objectForInfoDictionaryKey:@"FLTAssetsPath"];
```

Comment:
@chinmaygarde this mirrors what https://github.com/flutter/engine/blob/master/shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm#L158 is doing, but I could not find who actually initializes `FLTAssetsPath`. Do you know who does put it into the bundle?